integration_tests/tests/dbt_project.py

```diff
@@ -36,7 +44,22 @@ def get_dbt_runner(target: str) -> DbtRunner:
 
 class DbtProject:
     def __init__(self, target: str):
-        self.dbt_runner = get_dbt_runner(target)
+        self.dbt_project_copy_dir = Path(mkdtemp(prefix="integration_tests_project_"))
```

Comment:
im worried it might increase the runtime of the tests substantially, would you mind checking?
in case it does, maybe a copy per process/thread?